app/models/alert_date.py

```diff
@@ -57,6 +57,11 @@ def as_url(self):
     def as_lang(self, lang='en-GB'):
         return f'at {self.time_as_lang} on {self.date_as_lang}'
 
+    @property
+    def as_alert_lang(self, lang='en-GB'):
+        dt = self._local_datetime
+        return f'{dt:%A} {dt.day} {dt:%B} {dt:%Y}, at {dt:%-I%p}'
+
```

Comment:
Is there a test for this function?

Reply:
Have added a few assertions for as_alert_lang alongside the other alert_date tests.